Source/Stationeers.Addons.Patcher/Stationeers.Addons.Patcher.csproj

```diff
@@ -18,7 +18,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\Build\Patcher\Debug\</OutputPath>
```

Comment:
Ah this overwrite was not intended to be included in this PR

Reply:
Resolved: https://github.com/Erdroy/Stationeers.Addons/pull/3/commits/6a7422ba90e21c6ff3d86ba717c5b3e28314fb3b